org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserClassLoader.java

```diff
@@ -41,11 +47,13 @@ public UserClassLoader(URL[] urls, ClassLoader parent, List<String> userProvided
     if (this.allowedClasses.contains(name)) {
       return this.approvedClassLoader.loadClass(name);
     }
-    // allow .<specific-class> usage from valid packages
-    for (int i = 0; i < this.allowedPackages.length; i++) {
-      if (name.startsWith(this.allowedPackages[i])) {
-        return this.approvedClassLoader.loadClass(name);
-      }
+    // allow .<specific-class> usage from allowed packages. If this code has elevated permissions,
+    // also check the
+    // elevated permissions allowed package list.
```

Comment:
NIT: Spacing. (I think the linter made some odd choices here around line breaks.)

Reply:
yep I think this was the linter, will fix